packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2387,6 +2387,11 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// Updates the semantic information of the render object.
   void _updateSemantics() {
     assert(_semanticsConfiguration.isSemanticBoundary || parent is! RenderObject);
+    if (_needsLayout) {
+      // There's not enough information in this subtree to compute semantics.
+      // The subtree is probably currently offstage.
```

Comment:
offstage is the wrong word here, because the Offstage widget turns off painting but actually maintains layout.
Maybe "being kept alive by a viewport but not laid out" or something.